.github/workflows/firstCI.yml

```diff
@@ -9,44 +9,24 @@ on:
 jobs:
   build:
 
-    runs-on: macos-11
```

Comment:
No idea why, but setting up mysqld worked on macos and not on ubuntu.
By setting up, I mean executing `bin/mysqld --defaults-file=./my.cnf --daemonize`.
On ubuntu, mysqld initialization always returned error messages about error log and data file unusable.

Reply:
I see.
The default path of error logs in mysql is `/var/log/mysql/error.log`, and I guess github actions runner does not have access permissions to this directory.
If we specify the error-log directory path in `my.cnf` just like `log-error=./data/error.log`, the test would work on ubuntu.

However, the current goal is to pass the test regardless of the environment.
Thus now forget about this change.
I'll revert it.